h2o-core/src/main/java/water/api/NpsBinServlet.java

```diff
@@ -19,13 +20,14 @@
 
   @Override
   protected void doGet(HttpServletRequest request, HttpServletResponse response) {
-    String uri = JettyHTTPD.getDecodedUri(request);
+    JettyHTTPD jetty = H2O.getJetty();
```

Comment:
Not related to change: In theory, we should not need jetty instance inside servlet. Trying to figure out why we need it here.